build.rs

```diff
@@ -15,18 +15,23 @@
 // along with cargo-contract.  If not, see <http://www.gnu.org/licenses/>.
 
 use std::{
+    borrow::Cow,
     env,
     ffi::OsStr,
     fs::File,
     io::{prelude::*, Write},
     iter::Iterator,
     path::{Path, PathBuf},
+    process::Command,
 };
 
 use anyhow::Result;
+use platforms::*;
```

Comment:
I would prefer not having a wildcard import here.
```suggestion
use platforms::{TARGET_ARCH, TARGET_ENV, TARGET_OS};
```

Reply:
have fixed